src/pm/hydra/tools/bootstrap/external/slurm_query_node_list.c

```diff
@@ -125,7 +352,7 @@ static HYD_status list_to_nodes(char *str)
   fn_fail:
     goto fn_exit;
 }
-
+#endif
```

Comment:
Can you please add a newline after `#endif` and not immediately start the next function?

Reply:
Done

Reply:
Wrong patch.

Reply:
Sorry about that. Should be good now.